src/makeInfinityAdder.js

```diff
@@ -4,7 +4,22 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+    let sum = 0;
+
+  function adder(num) {
+    if (num === undefined) {
+      const newSum = sum;
+
+      sum = 0;
+
+      return newSum;
+    }
+    sum += num;
```

Comment:
The addition operation is correct. It adds the passed number to the current sum.